examples/vad_transcriber/requirements.txt

```diff
@@ -0,0 +1,4 @@
+deepspeech==0.2.0
+tensorflow==1.6.0
```

Comment:
@b-ak You don't need the `tensorflow` dependency

Reply:
It'll happen as part of the deepspeech install itself right ? So, no need to explicitly mention it..

Reply:
There will be tensorflow already statically linked with libdeepspeech.so, so no install at all